app/gateway/2.6.x/install-and-run/helm.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Install on Kubernetes with Helm
+---
+
+This page explains how to install {{site.base_gateway}} with {{site.kic_product_name}} using Helm. The Enterprise deployment includes a Postgres sub-chart provided by Bitnami. The documentation on installing with a [flat Kubernetes manifest](/gateway/{{page.kong_version}}/install-and-run/kubernetes) also explains how to install in DB-less mode for both Enterprise and OSS deployments.
```

Comment:
```suggestion
This page explains how to install {{site.base_gateway}} with {{site.kic_product_name}} using Helm.

* The Enterprise deployment includes a Postgres sub-chart provided by Bitnami. 
* For open-source deployments, you can choose to use the Postgres sub-chart, or install without a database. 

Configuration for both options is flexible and depends on your environment.

The documentation on installing with a [flat Kubernetes manifest](/gateway/{{page.kong_version}}/install-and-run/kubernetes) also explains how to install in DB-less mode for both Enterprise and OSS deployments.
```
How about something like this? I don't want to make it sound like the Postgres sub-chart is for Enterprise only, and I think it's useful to have a parallel for OSS.